File: snippets/ruby/view_a_contact_s_tags.rb
require "MailchimpMarketing"
require "digest"

mailchimp = MailchimpMarketing::Client.new
mailchimp.set_config({
  :api_key => ENV["API_KEY"],
  :server => ENV["API_SERVER"]
})
list_id = ENV["LIST_ID"]

email = "urist.mcvankab@example.com"
subscriber_hash = Digest::MD5.hexdigest email.downcase

response = mailchimp.lists.get_list_member_tags list_id, subscriber_hash

puts "Urist has been tagged #{response["total_items"]} times."

File: snippets/node/view_a_contact_s_tags.js
const listId = "YOUR_LIST_ID";
const email = "urist.mcvankab@example.com";
const subscriberHash = md5(email.toLowerCase());

async function run() {
  const response = await mailchimp.lists.getListMemberTags(listId, subscriberHash);

  console.log(`Urist has been tagged ${response.total_items} times.`);
}

run();